samples/SessionSample/Startup.cs

```diff
@@ -5,13 +5,31 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 
 namespace SessionSample
 {
     public class Startup
     {
+        // Uncomment the following line to use the Microsoft SQL Server implementation of IDistributedCache.
```

Comment:
Can't you leave all of this constructor uncommented? It doesn't change the app behavior. Only AddSqlServerCache needs to be commented out.